payments/notifications/dummy_context.py

```diff
@@ -101,6 +101,26 @@ def _get_winter_storage_order_context():
     return _get_order_context(order, [], optional_services)
 
 
+def _get_additional_product_order_context():
+    customer = CustomerProfileFactory.build()
+    order = OrderFactory.build(
+        customer=customer,
+        lease=BerthLeaseFactory.build(customer=customer),
+        product=None,
+        price=Decimal("0.00"),
+        tax_percentage=Decimal("0.00"),
```

Comment:
One thing to consider if we're gonna be setting the tax percentage to zero, is to update this to avoid division errors. Or, just stick to the predefined `PLACE_PRODUCT_TAX_PERCENTAGES`.

```python
    @property
    def total_tax_percentage(self):
        return (
            rounded_decimal(
                ((self.total_price - self.total_pretax_price) / self.total_pretax_price)
                * 100,
                round_to_nearest=0.05,
            )
            # Add this check
            if self.total_pretax_price != 0
            else 0
        )
```